scripts/get_event_schemas.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
+mkdir -p internal/app/testdata
+curl -o internal/app/testdata/application-updated.json "https://raw.githubusercontent.com/ministryofjustice/opg-event-store/main/domains/POAS/events/application-updated/schema.json"
+curl -o internal/app/testdata/reduced-fee-requested.json "https://raw.githubusercontent.com/ministryofjustice/opg-event-store/main/domains/POAS/events/reduced-fee-requested/schema.json"
```

Comment:
we could pull them each time the test is run, but I think having to manually update them will be less surprising

Reply:
This is cool for now - it may be worth thinking of some other way to enforce the schemas being up to date in some still undetermined way as we're only as safe as our brains remembering to run the script and commit.